osgar/drivers/kloubak.py

```diff
@@ -222,20 +285,50 @@ def update_buttons(self, data):
                 self.bus.publish('emergency_stop', self.emergency_stop)
                 print('Emergency STOP:', self.emergency_stop)
 
+    def partial_axis_encoders_update(self, arr):
+        # return True when completed
+        # simple report encoders on epoch change -> delayed 1 cycle
+        epoch, axis, left, right = arr
+        ret = self.partial_encoders[:]
+#        assert axis in [1, 2, 3], axis
+        if axis == 0:
+            axis += 1 # can error?
```

Comment:
NO NO NO! `axis==0` is definitely ERROR, so if we are not asserting it (I understand the reasons) I would print it and return `ret` without modification

Reply:
maybe `return None` would be more appropriate in such case

Reply:
I agree but I have to remind that it was your suggestion ;-).